lib/cn/cn.c

```diff
@@ -220,6 +220,9 @@ cn_get_perfc(struct cn *cn, enum cn_action action)
 
     case CN_ACTION_SPLIT:
         return &cn->cn_pc_split;
+
+    case CN_ACTION_JOIN:
+        return &cn->cn_pc_split;
```

Comment:
@nabeelmmd can you fix this in your join patch please?

Reply:
Sure, will do!